org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -272,6 +275,11 @@ public void dispose() {
 
             // Shut down the ZigBee library
             networkManager.shutdown();
+
+            // If the bridge has been removed then delete the network state file
```

Comment:
Suggestion: Remove the comment - as it basically just says what the following three lines of code say. What do you think?

Reply:
OK. I think the code is clear enough.